openpype/hosts/maya/api/lib.py

```diff
@@ -174,6 +174,22 @@ def maintained_selection():
             cmds.select(clear=True)
 
 
+def reload_textures(preset):
+    """Reload textures during playblast
+    """
+    if not preset["viewport_options"]["reloadTextures"]:
+        self.log.debug("Reload Textures during playblasting is disabled.")
+        return
+
+    texture_files = cmds.ls(type="file")
+    if not texture_files:
+        return
```

Comment:
Are we sure reloading textures is only ever needed for `file` nodes? how about `aiImage` or other types of files?